src/extension.ts

```diff
@@ -106,27 +106,41 @@ function fetchAndLaunchMetals(context: ExtensionContext, javaHome: string) {
     p => !p.startsWith("-agentlib")
   );
 
+  const customRepositories: string[] = config
+    .get<string>("customRepositories")!
+    .toString()
+    .split(" ")
```

Comment:
See my other comment, if we make the configuration an array, we can do

```ts
const customRepositories: string[] = config.get<string[]>("customRepositories");
```

instead